plugins/LayerView/LayerPass.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2015 Ultimaker B.V.
```

Comment:
Happy new year! I've updated this for you in 9e4b7e29ccec1cfb76ef0ac87b65bf4c7ca840b1.